src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -324,21 +258,24 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   @Getter(useStore) getEntitySnapshot!: EntitySnapshotIF
   @Getter(useStore) getEntityType!: CorpTypeCd
   @Getter(useStore) getResource!: ResourceIF
+  @Getter(useStore) hasBusinessNameChanged!: boolean
   @Getter(useStore) hasBusinessTypeChanged!: boolean
   @Getter(useStore) isBcCompany!: boolean
+  @Getter(useStore) isBenefitCompany!: boolean
+  @Getter(useStore) isBcUlcCompany!: boolean
   @Getter(useStore) isConflictingLegalType!: boolean
+  @Getter(useStore) isNumberedCompany!: boolean
 
   @Action(useStore) setEntityType!: ActionBindingIF
+  @Action(useStore) setNameRequest!: ActionBindingIF
+  @Action(useStore) setNameChangedByType!: ActionBindingIF
 
-  protected selectedEntityType = null as CorpTypeCd
+  selectedEntityType = null as CorpTypeCd
   protected confirmArticles = false
```

Comment:
Don't need `protected`, please remove for these lines

Reply:
It's originally protected there before, I can just remove it, but can't use readonly cause property's value needs to be changed during lifecycle.